frontend/src/components/Item/ItemPreviewBox.tsx

```diff
@@ -20,15 +21,17 @@ export type ItemPreviewProps = {
  * 
  * Example:
  * ```tsx
- * <ItemPreview id="1" name="Smartphone" weight="300 gr" imageUrl="https://via.placeholder.com/96.png" />
+ * <ItemPreviewBox id="1" name="Smartphone" weight="300 gr" imageUrl="https://via.placeholder.com/96.png" />
  * ```
  */
-export const ItemPreview: React.FC<ItemPreviewProps> = ({ slug, datacy, name, weight, imageUrl }) => {
+export const ItemPreviewBox: React.FC<ItemPreviewProps> = ({ slug, datacy, name, weight, imageUrl }) => {
+    const weightString = `${weight.isCa ? "ca." : ""}${weight.value}${weight.aditionalValue ? `- ${weight.aditionalValue}` : ""} g`
```

Comment:
1. Fallback value for weight.value => Sonst würde da null stehen so wie ich das sehe würde da aktuell null stehen können
2. Nur `g` als Angabe ist suboptimal. Ich hätte ich dem Fall behauptet, dass sich an Wikipedia orientieren das sinnigste ist. Die schreiben Gramm aus und haben ein Leerzeichen zwischen Wert und Einheit ( siehe [Beispiel](https://en.wikipedia.org/wiki/Gram))

Reply:
1. True my bad

2. Wenn du meinst, wollte es nur mal angemerkt haben